bolt.gemspec

```diff
@@ -44,7 +44,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "logging", "~> 2.2"
   spec.add_dependency "minitar", "~> 0.6"
   spec.add_dependency "net-scp", "~> 1.2"
-  spec.add_dependency "net-ssh", ">= 4.0"
+  spec.add_dependency "net-ssh", "~> 5.2"
```

Comment:
I'm not sure if this is still a thing w/ beaker https://github.com/puppetlabs/bolt/blob/c57dd5663145358da4d919900e53dd789637bdfb/lib/bolt/transport/ssh/connection.rb#L60-L61 but we were at one point trying to keep compatabilty w/ netssh 4.